Server/models/order.js

```diff
@@ -0,0 +1,75 @@
+const mongoose = require('mongoose');
+
+const orderItemSchema = new mongoose.Schema(
+    {
+        name: {type: String, required: true},
+        qty: {type: Number, required: true},
+        image: {type: String, required: true},
+        price: {type: Number, required: true},
```

Comment:
I don't think we need image and price here - just product's id as we'd be able to pull those ourselves, or a ref to product